lib/src/dart.dart

```diff
@@ -8,116 +8,13 @@ import 'dart:io';
 import 'dart:isolate';
```

Comment:
It looks like `isPart()` in this file is no longer used.

Reply:
Removed along with `runInIsolate`.